packages/contracts-bedrock/contracts/L1/L1CrossDomainMessenger.sol

```diff
@@ -43,12 +44,17 @@ contract L1CrossDomainMessenger is CrossDomainMessenger, Semver {
      * @inheritdoc CrossDomainMessenger
      */
     function _sendMessage(
+        uint32 _type,
         address _to,
         uint64 _gasLimit,
         uint256 _value,
         bytes memory _data
     ) internal override {
-        PORTAL.depositTransaction{ value: _value }(_to, _value, _gasLimit, false, _data);
+        if (_type == BridgeConstants.BIT_TX){
+            PORTAL.depositTransaction{ value: 0 }(_type,_to, _value, _gasLimit, false, _data);
+        }else{
+            PORTAL.depositTransaction{ value: _value }(_type,_to, _value, _gasLimit, false, _data);
+        }
```

Comment:
For eth deposit, the value will be assigned to zero here. The deposit ETH should be locked in PORTAL. If we forcefully assign zero to value, I think there will be a problem here.